src/applications/personalization/profile/components/ProfileInfoTable.jsx

```diff
@@ -17,6 +18,7 @@ const ProfileInfoTable = ({
   title,
   className,
   namedAnchor,
+  level = 3, // heading level
```

Comment:
Can you add this to the prop types? Extra credit for making sure it’s a number from 1-6

Reply:
done